### PR TITLE
595 add function to override header font size and kerning in collections and exhibits

### LINKED ISSUE #595: Add function to override header font size and kerning in Collections and Exhibits
SCRC staff would like advanced controls for header text in Collections and Exhibits to allow for fonts to be resized or spacing to be adjusted.

### PR DESCRIPTION
**Pull request by bluairis**
Fixes #595 

**Changes:**
- Added the ability to customize the font size and kerning in customizable web exhibits like [the soviet imaginary page](https://www.lib.uchicago.edu/collex/exhibits/soviet-imaginary/)
- Font size:
> float value (*note: allows users to increment by 1 on the right...but this behavior might not be the same in the new wagtail)
> allows no input value and then defaults to 16.8 px, the standard font size for collections/exhibits (*note: also defaults to 16.8 for inputs of 1, 0 and negative inputs)
> inputs are multiplication factors that multiply by 16.8 px to determine the font size
> description might need tweaking

- Font kerning:
> float value (*note: allows users to increment by 1 on the right...but this behavior might not be the same in the new wagtail)
> allows no input value and then defaults to 0 px, which indicates no kerning besides that of the actual font (*note: negative values bring the font closer together)
> description might need tweaking

**To Make Actual Changes Easier to Spot** (and not the changes from the linter)
In the `models.py`:
1. added default values on lines 43 and 44
2. defined the fields for font_scaler and font_kerning from line 2020 to line 2029
3. added the above fields to the Field Panel on lines 2049 and 2050
4. defined when to use default values and when to use inputed values from line 2288 to 2295
5. added the fields to the context on lines 2322 and 2323

### CODE REVIEW

**Review by bbusenius — CHANGES_REQUESTED** [review]
For some reason I thought you were developing off the Wagtail 4.x branch (and didn't notice that you weren't in the meeting). That's OK but I'm happy that you haven't added migrations and fixtures to this pull request. Let's hold off on that until we're ready to merge with the new Wagtail.

This looks good. My only suggestion would be to work with @dbietila on the help text for the new fields.

I'm noticing that you still have some issues with linting. When I run the file through my linter, I get a different output. I'm not worried about that for this pull request. I think we all need to sync up. When we move back to Black, would probably be the right time for that.

Lastly, I'm unable to build this. I think that after we merge this with the `master` branch and make migrations / fixtures, it will build fine though. I'm not worried about that.

Before we merge this into `master`, we need to do the following in order:

1. Merge `master` into this branch.
2. Make migrations and build fixtures after merging `master`.
3. Make sure it builds and passes tests.

After the above steps we can merge feature branch with `master`.

_Updated this comment to reflect post-upgrade reality._

**Comment by bufordrat** [comment]
> For some reason I thought you were developing off the Wagtail 4.x branch (and didn't notice that you weren't in the meeting). That's OK but I'm happy that you haven't added migrations and fixtures to this pull request. Let's hold off on that until we're ready to merge with the new Wagtail.

HA.  I was just going to comment "where are the migrations"?  Somehow I also failed to notice we were on the old version of Wagtail during the meeting.

**Pull request closed without merging**
